recipes/sdl2_mixer/CmakeLists.txt

```diff
@@ -0,0 +1,236 @@
+# Written by Daniel Schreij (dschreij@gmail.com) on 1 Nov 2016
+# This procedure is configured to work in an Anaconda build environment (contiuum.io)
```

Comment:
Probably by using mercurial ;) but I'll give it a go. I think they are working on this themselves too. All sdl2 modules (with exception of sdl2_gfx) are developed by the core sdl2 team, as far as I know, and sdl2 has a CMakeLists.txt packaged with it, so I don't think it will be long before it is included in with the source of all modules as well.